content-security-policy/script-src/scripthash-case-insensitive.sub.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta http-equiv="Content-Security-Policy" content="script-src 'self' 'nonce-abc' 'SHA256-VCOfB9NQbtW8/s+T7yizqn0dz0Ipt5krwH9BPUaXJTA=' 'SHA384-efOmACJwOYjUewZJTpktK4Kxl9spgncVwxok9DaIBIMN2zBzwxDni19L5uHkIX3E' 'SHA512-t9CmeiAGRym+Wsi8F+5TV1QEjcbFppf7ONB9HUTOs5pMLUy3BQCmASwXD/VKl0B5QytTTJawA2IhVvoebs7Gyg==' 'sHa256-BPe1cNQpEQoucXTYM91Ku9xnHT/BZXMOeOFeMZTPWis=' 'shA384-qNmIi2ya4g29IbFyUBBPFJ5BdkW43bygT/MrFSoe7o/ALn+a3iJDkssigmMHQ4J0' 'Sha512-GuQbQFeVHDBySntDnOpbrNCe4xwjLhnnaVRAGz5JAnYK9pj0vOEAkmKgzNJApgufV3r37DE7Derx5DGUmqkukg=='; connect-src 'self';">
+    <title>scripthash-case-insensitive</title>
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/content-security-policy/support/alertAssert.sub.js?alerts=%5B%22PASS%20(1%2F6)%22%2C%22PASS%20(2%2F6)%22%2C%22PASS%20(3%2F6)%22%2C%22PASS%20(4%2F6)%22%2C%22PASS%20(5%2F6)%22%2C%22PASS%20(6%2F6)%22%5D">
+    <script>
```

Comment:
> I guess this script should be allowed by some of the CSP source expressions above, but I don't think it is.

hmm… I guess it gets special-cased?

Reply:
What do you mean? I guess it is just blocked. I suggest you a nonce and allow it.

Reply:
> What do you mean? I guess it is just blocked. I suggest you a nonce and allow it.

OK, added a nonce for it.

What had confused me is: No CSP violation was being reported in the devtools console. But after looking at the DOM, I realized the reason is, the preceding `<script src>` element had no `</script>` end tag – so the following `<script>` start tag and that JavaScript were just getting parsed as text content for the preceding `script` element. And since that `script` element has `src` attribute, the browser just ignores the text content of that `script`.

So I added a `</script>` end tag to the preceding `script` — and then, as I said, added a nonce to the following one.

So I think we’re all good now on that part.